src/components/Comments.jsx

```diff
@@ -0,0 +1,48 @@
+import { useContext, useEffect, useState } from "react";
+import { UserContext, UserProvider } from "../contexts/UserContext";
+import { getAllArticles, getCommentsByArticleID } from "../utils/api";
+import { Link } from "react-router-dom";
+import { useParams } from "react-router-dom";
+
+function Comments() {
+  const [comments, setComments] = useState("");
```

Comment:
It is best practice to have the state set to an empty version of what we expect it to be set to later down the line, I think comments will be an array rather than a string